README.md

```diff
@@ -1,88 +1,97 @@
 
-### Important Note
+## Requirements
 
-Building and running the TrueBlocks docker image is taken care of by the dAppNode package manager. The following instructions are presented for those who are not running dAppNode. If you are running a dAppNode, you can simply install the package from the dAppNode package manager.
+- A running, syncing Parity Ethereum node with `--tracing on` enabled (see [Running A Node](#how-do-you-recommned-I-run-a-node))
 
-### Building the Docker Image
+- Time (see [Why Does It Take So Long To Build The Index?](#why-does-it-take-so-long-to-build-the-index))
 
-Make a local copy of the TrueBlocks docker repo on your machine:
+- git, docker, docker-compose (see [Prerequisites](#prerequisites))
 
-```
-$ git clone https://github.com/Great-Hill-Corporation/DAppNodePackage_trueblocks.git
-```
+## Installation
 
-Build the docker image:
+The following instructions are for building on your local machine. Running on DappNode? See our [Dappnode Instructions](#dappnode-instructions).
 
 ```
-$ docker-compose build
+git clone https://github.com/Great-Hill-Corporation/trueblocks-docker.git
+cd trueblocks-docker
+docker-compose build
 ```
-or, alternatively:
 
-```
-$ docker build --rm -f build/Dockerfile -t dnp_trueblocks:dev build 
-```
+## Before running
+
+Before you bring the TrueBlocks docker container up, take the time to configure the following options:
+- what port do you want to run the trueblocks api server on? Set this in docker-compose.yml. Right now, the ports line reads 80:80. If you want to change this from port 80 to, say, 8181, then change this to 8181:80.
+- What is your node's RPC endpoint? For some users, this is http://localhost:8545, but due to Docker's approach to networking, this is probably not the address you want to use (see [What is my RPC endpoint?](#what-is-my-rpc-endpoint)). Whatever the case, set your RPC endpoint in the trueblocks.public.dappnode.eth.env file to `RPC_PROVIDER=http://your-rpc-provider:port`.
+- Now you're ready to run trueblocks!
+
+## First run
 
-## Running
-### Starting Parity
-In order for TrueBlocks to work properly, you need to start parity with at least the following options:
+Bring the container up:
 
 ```
-parity --tracing on --jsonrpc-cors all --jsonrpc-hosts all --db-compaction=ssd 
+docker-compose up -d
 ```
 
-`--db-compaction=ssd` is optional, but Parity recommends using an SSD drive for storing its data. Enabling `--tracing on` requires a re-sync of your node.
+There are 2 modes of operation for trueblocks: scrape and leech.
 
-We also recommend (although, this is optional) that you specify an alternative path to store Parity's data (this will also require a re-sync of your node). We suggest an external 4TB SSD.
+- **scrape**: scrape the entire Ethereum chain and build the TrueBlocks address index from scratch.
+- **leech**: instead of building the TrueBlocks address index, download it from a 3rd party.
```

Comment:
Don't use the words "third party" here. The idea here is that they can download it from "a billion third parties none of which has the power to refuse effectively because none of them has a monopoly on the data." Also, in the header of the data is the hash of the git commit that built the data so if they want to they can re-create the data on the fly. There more to it than "download from a third party". It's more like download data that has been created under the scientific method. We've included the exact instructions you will need to re-create this data yourself. Understand that you're using this data because you chose convenience instead of security. But, unlike any other data you'll find, we've included the instructions and we've published the data on IPFS so we can't keep you from getting it. The more the community uses the data, the less possible it becomes for any one entity to capture it."

Reply:
A lot of work needs to go into this section in general. I'm going to leave this as it is, and follow up in #37.